### PR TITLE
Added report of node-killer issue to Known Issues.

### PR DESCRIPTION
**Pull request by markgbeckett**

### CODE REVIEW

File: docs/known-issues/index.md [hunk 1]
```diff
@@ -6,6 +6,17 @@ active investigation by HPE Cray and the wider service.
...
 
+### Occasionally user jobs can cause compute nodes to crash (Added: 2021-11-22)
+
+- **Systems affected:** ARCHER2 full system
+
+In rare circumstances, it is possible for a user job to crash the compute nodes on which it is running. This is only evident to the user as a failed job: there is no obvious sign that the nodes have crashed. Therefore, if we identify a user whose jobs are causing nodes to crash, we may need to work with them to stop this happening.
+
+The underlying issue is resolved in an update to the compute-node operating system (Shasta Version 1.5), which is expected to be rolled out to the main system early in 2022.
+
+In the meantime, users who experience this issue are advised to try disabling XPMEM in their application. CSE can provide advice on how to do this.
```
Comment (aturner-epcc) [inline docs/known-issues/index.md:17]: @markgbeckett Can you add note on how to contact them here. e.g.

You can contact the ARCHER2 CSE team at EPCC via [the service desk](https://www.archer2.ac.uk/support-access/servicedesk.html).

**Review by aturner-epcc — CHANGES_REQUESTED** [review]
Thanks @markgbeckett Just a request to add details on how to contact the service desk.

File: docs/known-issues/index.md (hunk 1 above)
Comment (markgbeckett) [reply docs/known-issues/index.md]: Hi @aturner-epcc , I've made the change you suggested. Can you recheck? George.

**Review by aturner-epcc — APPROVED** [review]
Thanks @markgbeckett I will merge this now

**Pull request merged**